src/OSPSuite.Core/Domain/Formulas/Formula.cs

```diff
@@ -69,27 +68,20 @@ public interface IFormula : IObjectBase, IWithDimension
 
    public abstract class Formula : ObjectBase, IFormula
    {
-      private readonly List<IObjectReference> _objectReferences;
-      private List<IFormulaUsablePath> _objectPaths;
-      public virtual IDimension Dimension { get; set; }
+      private readonly List<IObjectReference> _objectReferences = new List<IObjectReference>();
+      private List<IFormulaUsablePath> _objectPaths = new List<IFormulaUsablePath>();
+      public virtual IDimension Dimension { get; set; } = Constants.Dimension.NO_DIMENSION;
```

Comment:
For Formula: Dimension should be initialized when created. Same with YDimension for TableFormula. This is becoming a problem with my pH table having no Dimension when being deserialized